de.tubs.variantsync.core/src/de/tubs/variantsync/core/patch/base/DefaultMarkerHandler.java

```diff
@@ -25,7 +25,8 @@
 		List<IVariantSyncMarker> variantSyncMarkers = new ArrayList<>();
 		for (IDelta<Chunk<String>> delta : deltas) {
 			Chunk revised = delta.getRevised();
-			IVariantSyncMarker variantSyncMarker = new AMarkerInformation(revised.getPosition() - 1, revised.getLines().size() - 1, true);
+			IVariantSyncMarker variantSyncMarker = new AMarkerInformation(revised.getPosition() - 1,
+					revised.getLines().size() - 1, true);
```

Comment:
Explain -1 in comment.

Reply:
This problem will be addressed by the following issue https://github.com/tthuem/VariantSync/issues/56